cli/cmd/logs.go

```diff
@@ -10,13 +12,31 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var logsDescription = "Check Service logs. Requires one Service <node,electrs,esplora,chopsticks>"
```

Comment:
Never repeat yourself. A function helper should return the strings that indicate the services, so if we add a service we change in a single place, not everywhere scattered around.